kubernetes/Dockerfile

```
@@ -12,6 +12,8 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s
 RUN groupadd -g 12000 -r checkov && useradd -u 12000 --no-log-init -r -g checkov checkov
 RUN mkdir /data && mkdir /app && mkdir /home/checkov
 RUN chown checkov:checkov /data /app /home/checkov
+RUN curl -L -o /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.16.2/yq_linux_amd64 \

```

```suggestion
RUN curl -L -o /usr/bin/yq https://github.com/bridgecrewio/yq/releases/download/v4.16.2/yq_linux_amd64 \
```

I've forked it to our org for stability (in case original is deleted) and security (prevent injection of unexpected new code)